crypto/evp/p_seal.c

```diff
@@ -31,21 +32,27 @@ int EVP_SealInit(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *type,
         return 1;
     if (EVP_CIPHER_CTX_rand_key(ctx, key) <= 0)
         return 0;
+
     if (EVP_CIPHER_CTX_iv_length(ctx)
-        && RAND_bytes(iv, EVP_CIPHER_CTX_iv_length(ctx)) <= 0)
-        return 0;
+            && RAND_bytes(iv, EVP_CIPHER_CTX_iv_length(ctx)) <= 0)
```

Comment:
Styling nit: extra indentation

Reply:
This doesn't look like an extra indent to me -- I thought we'd special cased an extra indent after `if` to separate the compound conditional expression from the following code.  Or have I missed something obvious?

Reply:
That is the pattern used in other places. i.e indent the next line in so that it doesnt match the ending block (i,e return or goto).

Reply:
The extra indent for if makes sense, but I couldn't find it in the coding style guidelines.

@mattcaswell took care of the restyling, maybe he can comment on this?

Reply:
It isn't in the coding style guidelines, it's true (there may be a proposal to change this, I'll have to check again).  This extra indentation is *optional* but acceptable, at the leasure of the author.

Reply:
As @paulidale says this is to separate the conditional from the start of the following code. This isn't universally followed by everyone and as @levitte says it isn't in the style guide but is generally accepted as ok style.

Reply:
Perfect, I wasn't aware this was accepted (I actually prefer to use this extra indent, but refrained from using it until now) !

Looks like all the required approvals are in place. 

I can take care of merging!